app/views/admin/checks/index.html.erb

```diff
@@ -30,6 +33,9 @@
             <span class="app-task-list__task-name">
               <%= link_to "Check employment information", admin_claim_check_path(claim_id: @claim.id, check: :employment), class: "govuk-link" %>
             </span>
+            <% if @claim.checks.exists?(name: "employment") %>
+              <strong class="govuk-tag app-task-list__task-completed" id="eligibility-completed">Completed</strong>
```

Comment:
There's an ID attribute on this tag. Any particular reason?

Reply:
Ah, I mistakenly copied it from the prototype. I removed it from the one above but forgot this one. I'll remove it now